mechanisms/mod/test_kinlva.mod

```diff
@@ -0,0 +1,76 @@
+: Adaption of T-type calcium channel from Wang, X. J. et al. 1991;
+: c.f. NMODL file in ModelDB:
+: https://senselab.med.yale.edu/modeldb/showModel.cshtml?model=53893
+:
+: Note the temperature rate correction factors of 5 (for m) and 3
+: (for h <-> s <-> d) have been applied to match the model described
+: in the current-clamp experiments (see p. 842).
+
```

Comment:
The channel implementation is _ex novo_, derived directly from the equations and parameter values from the paper. The NMODL file on modelDB is just linked for comparison; the reference could be removed.